py4DSTEM/process/diffraction/crystal.py

```diff
@@ -660,6 +662,8 @@ def orientation_plan(
             angle_step_in_plane (float):  Approximate angular step size for in-plane rotation [degrees]
             accel_voltage (float):        Accelerating voltage for electrons [Volts]
             corr_kernel_size (float):        Correlation kernel size length in Angstroms
+            radial_power (float):          Power for scaling the correlation intensity of each radial bin
+            intensity_power (float):       Power for scaling the intensity of structure factors
```

Comment:
I think these docstrings could be a touch clearer:
Power for scaling the correlation intensity as a function of the peak radius
Power for scaling the correlation intensity as a function of the peak intensity